addons/binding/org.openhab.binding.networkcamera/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="networkcamera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="motiondetection">
+		<label>Motion Detection</label>
+		<description>Receive images from network camera.</description>
+
+		<channels>
+			<channel id="image" typeId="image-channel" />
+			<channel id="motion" typeId="motion-channel" />
+			<channel id="motion-trigger" typeId="motion-trigger" />
```

Comment:
I think this channel is also kind of redundant, because you can obtain this information in rules by creating a condition whether the state of the item on the image channel has been changed, right?